src/test/java/org/stellar/sdk/TransactionTest.java

```diff
@@ -159,6 +163,7 @@ public void testToBase64EnvelopeXdrBuilderNoSignatures() throws FormatException,
             },
             null,
             new TransactionPreconditions(null, null, 0, 0, new ArrayList<SignerKey>(), null),
+            Optional.absent(),
```

Comment:
should there be at least one test of a Transaction that has SorobanTransactionData?

Reply:
I prefer to maintain consistency with [the definition in Horizon](https://github.com/stellar/go/blob/soroban-xdr-next/protocols/horizon/operations/main.go#L362). Perhaps we should modify the definition in Horizon to `functionType`?

Reply:
Fixed in c0d2163fac4f72c7a5e520d97ca398dadc2c167d